core/chains/evm/txmgr/orm.go

```diff
@@ -84,31 +90,26 @@ func (o *orm) preloadTxAttempts(txs []EthTx) error {
 	return nil
 }
 
-func (o *orm) preloadTxes(attempts []EthTxAttempt) error {
-	var ids []int64
+func (o *orm) PreloadEthTxes(attempts []EthTxAttempt) error {
```

Comment:
i overwrote this method with the code from loadEthTxes because it was more efficient

Reply:
Please be careful to make sure it is well tested, I wouldn't rule out subtle differences in behaviour.